bindings/experimental/distrdf/test/test_proxy.py

```diff
@@ -272,3 +272,11 @@ def test_getcolumnnames_after_define(self):
         cols_after = proxy.GetColumnNames()
 
         self.assertSequenceEqual(cols_after, ["x", "y", "z"])
+        
+    def test_get_column_type(self):
+        
+        hn = create_dummy_headnode(self.test_treename, self.test_filename)
+        proxy = Proxy.NodeProxy(hn)
+        column_type = proxy.GetColumnType("myColumn")
+        self.assertSequenceEqual(column_type, "Float_t")
+    
```

Comment:
We should add another test with a column created with `Define`

Reply:
yes, added now.